crates/telio-dns/tests/integration_tests.rs

```diff
@@ -10,35 +10,23 @@ use std::{
 };
 use telio_dns::{LocalNameServer, NameServer, Records};
 use tokio::net::UdpSocket;
-
-/// Wireguard key pair.
-struct KeyPair {
-    private: String,
-    public: String,
-}
-
-impl KeyPair {
-    /// Generate WireGuard key pair.
-    fn new() -> Self {
-        let private_key = X25519SecretKey::new();
-        let public_key = private_key.public_key();
-        KeyPair {
-            private: encode(private_key.as_bytes()),
-            public: encode(public_key.as_bytes()),
-        }
-    }
-}
+use x25519_dalek::{PublicKey, StaticSecret};
 
 /// Config is in INI format.
 struct WireguardConfig {
-    private_key: String,
-    peer_public_key: String,
+    private_key: StaticSecret,
```

Comment:
Always use telio_crypto definitions for our code.

Reply:
Done